### ISSUE TITLE
Avalon 7.5 Release Testing

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#4945]
# Description
Testing for the Avalon 7.5 release. 

# Testing
- [x] Regression Testing with CDL off -- started by @elynema see comment
- [x] CDL Functionality Testing with CDL enabled @joncameron @elynema
- [x] Playlists @masaball -- Found a number of bugs and created tickets for them.
- [x] Batch ingest and file upload @cjcolvar 
- [x] Bulk actions @Dananji -- In `Delete items in bulk` test case, I encountered a slightly different issue to the one detailed in the document. I made a comment in the doc.
- [x] Test item creation for objects of varying size @joncameron 
- [x] Text and video encoding testing (UTF-8, codecs) @joncameron 
- [x] Media player @Dananji 
- [x] Embedded Player @masaball 
- [x] Test login redirects - embedded player with restricted content, playlist, LTI login - on mco-staging and canvas-staging

**Comment by Dananji** [comment]
In `Delete items in bulk` test case, I encountered a slightly different issue to the one detailed in the document. I made a comment in the doc.

**Comment by cjcolvar** [comment]
Demo batch ingest manfiest rows with skip transcoding fail with minio dropbox:
```
The following rows of your spreadsheet failed:
Row	Error
4 	undefined method `close' for #<Addressable::URI:0x0000562fc2cc0bf8> Did you mean? clone
5 	undefined method `close' for #<Addressable::URI:0x00007f236d479158> Did you mean? clone 
```

**Comment by masaball** [comment]
Notes on playlist testing: https://docs.google.com/document/d/1Pa4gfEFDGe5lBz2zFQp_8HgSbof2cV1SoEaWfqUNUI4/edit?usp=sharing

**Comment by elynema** [comment]
I attempted regression testing on 11/22 on avalon-staging. It appeared that CDL was completely disabled on avalon-staging, as there was no option to turn it on for a collection. My notes are here. There were a few items I wasn't sure how to test. I found a potential bug under access control (but one that folks are extremely unlikely to ever hit). I'm not very familiar with Timeliner functionality, so I couldn't tell if the bubbles were working as they should be.

https://docs.google.com/document/d/1iCgB4jrwjuevhmR-JJ70Z-evcdcoHNdxe7Rhlfe0avU/edit?usp=sharing
